src/query/db_accessor.hpp

```diff
@@ -316,6 +316,10 @@ class DbAccessor final {
 
   void FinalizeTransaction() { accessor_->FinalizeTransaction(); }
 
+  std::atomic<bool> *TransactionAbortedByUser() { return accessor_->TransactionAbortedByUser(); }
+
+  bool IsTransactionActive() { return accessor_->IsTransactionActive(); }
```

Comment:
You need to extend also SubgraphDBAccessor with these two functions. SubgraphDBAccessor is used in case when we use `project` and in that case we will call from mg_procedure_impl IsTransactionActive